dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/SaveContentAsDraftActionlet.java

```diff
@@ -29,16 +31,27 @@
 public class SaveContentAsDraftActionlet extends WorkFlowActionlet {
 
 	private final ContentletAPI   contentletAPI;
-	private final RelationshipAPI relationshipAPI;
 	private final CategoryAPI     categoryAPI;
 	private final PermissionAPI   permissionAPI;
 
 	public SaveContentAsDraftActionlet() {
+		this(
+			APILocator.getContentletAPI(),
+			APILocator.getCategoryAPI(),
+			APILocator.getPermissionAPI()
+		);
+	}
+
+	@VisibleForTesting
+	public SaveContentAsDraftActionlet(
```

Comment:
I think this constructor doesn't need to be public

Reply:
done https://github.com/dotCMS/core/pull/18300/commits/1aa3444ac01b694697c3580a424d7e3566aabc35#diff-013f41c9cebd179a7239798b2e45fb94R46